enums.h

```diff
@@ -2316,6 +2316,7 @@ typedef enum spell_id {
 	SPL_FIREBALL    = 0xC,
 	SPL_GUARDIAN    = 0xD,
 	SPL_CHAIN       = 0xE,
+	SPL_SEARCH      = 0xE,

```

Comment:
Does chain lightning still work in Hellfire, or was it replaced with search?

Reply:
I belive it is still there, I just have to figure out why it appears to ocupy the same idu(probably lightning changed Id.)